api/src/services/stardust/influx/influxQueries.ts

```diff
@@ -6,6 +6,7 @@ export const BLOCK_DAILY_QUERY = {
             sum("tagged_data_count") AS "taggedData",
             sum("no_payload_count") AS "noPayload"
         FROM "stardust_block_activity"
+        WHERE time < now() - 1d
```

Comment:
`- 1d` in not good enough, as it depends when `now()` is computed. i.e. if computed at 11AM we will get the data up until 11AM of yesterday.
What we want is 12PM of yesterday always.

Additionally remember to update to `to` of the `parameterized` versions of the query also to be "12PM of previous day"